src/SpikeCore/SpikeCore/Modules/FactoidModule.cs

```diff
@@ -56,31 +55,35 @@ protected override async Task HandleMessageAsyncInternal(IrcPrivMessage request,
                 // Otherwise we're looking up factoids by name/type.
                 else
                 {
-                    // Grab the total count so we can tell users if there's more facts they can't see. They'll have to
-                    // hit the web UI for these.
-                    var count = _context.Factoids
-                        .Count(factoid => FactoidMatches(factoid, command, name));                   
+                    var query = _context.Factoids
+                        .Where(factoid => factoid.Type.ToLower() == command.ToLower() && factoid.Name.ToLower() == name.ToLower());
 
+                    var matchingFactoids = query
```

Comment:
Yeah, I guess this is probably cleaner than the copy pasta we ported over from the last bot. Thanks for the refactor.